src/client/components/CommunicationCard.tsx

```diff
@@ -116,17 +117,20 @@ const ieFlexOverflowFix = css`
 
 export const CommunicationCard: FunctionComponent<CommunicationCardProps> = ({
   title,
+  titleLevel = 3,
   body,
   value,
   image,
   checked,
 }) => {
+  const HeadingTag = `h${titleLevel}` as const;
+
   return (
     <div css={[greyBorderTop, blueBorderBottom, ieFlexOverflowFix]}>
       <div css={[communicationCardInfoContainer]}>
         <img css={img} src={image} alt={title} />
         <div css={ieFlexOverflowFix}>
-          <h3 css={communicationCardHeadingText}>{title}</h3>
+          <HeadingTag css={communicationCardHeadingText}>{title}</HeadingTag>
```

Comment:
I had no idea you could do this.. amazing!

Reply:
I've used strings as tags before, just needed to figure out how to make tsc happy (the `as const` did the trick).